test/utils.js

```diff
@@ -81,9 +90,165 @@ async function expectEvent(builder, keypair, eventData = {}) {
     }
 }
 
+function hashMessage(data, commandType) {
+    const toHash = new Uint8Array(data.length + 1);
+    toHash[0] = commandType;
+    toHash.set(data, 1);
+
+    return keccak256(toHash);
+}
+
+function signMessage(privKeys, messageToSign) {
+    const signatures = [];
+
+    for (const privKey of privKeys) {
+        const { signature, recid } = secp256k1.ecdsaSign(arrayify(keccak256(messageToSign)), arrayify(privKey));
+        signatures.push(new Uint8Array([...signature, recid]));
+    }
+
+    return signatures;
+}
+
+async function approveContractCall(client, keypair, gatewayInfo, contractCallInfo) {
```

Comment:
```suggestion
async function approveMessage(client, keypair, gatewayInfo, message) {
```
we switched to Message terminology instead of ContractCall